.gitignore

```diff
@@ -1,18 +1,15 @@
-.Rapp.history
-dist/
-node_modules/
-vendor/
-tmp/
-/*workspace
+.DS_Store
+.rehearsal.json
```

Comment:
we have mocks where we need to modify checkin this file. which is the exception paths we listed